### PR TITLE
Create profile for iamr-kumar

### PR DESCRIPTION
**Pull request by iamr-kumar**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

image
image

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2480"></a>

### CODE REVIEW

**Review by senali-d — COMMENTED** [review]
Look Good!

Happy Coding!

File: data/iamr-kumar.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "First Hackathon",
+      "date": "October 2020",
+      "icon": "FaCode",
+      "color": "blue",
+      "description": "Participated in my first hackathon and won it to got an internship offer",
+      "url": ""
```
Comment (Panquesito7) [inline data/iamr-kumar.json:52]: I don't think you should specify an URL if there's none. AFAIK, it could lead to an unwanted page.

Comment (iamr-kumar) [reply data/iamr-kumar.json:52]: Oh I though the url field is required but can leave empty. Will fix it.

**Review by Panquesito7 — APPROVED** [review]
Nice! Thanks. 🎉

**Pull request merged**